news/2017-12-14-OWC-semifinals-recap.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "osu! World Cup 2017 Semifinals Recap"
+date: 2017-12-14 10:00:00 +0000
+---
+
+Saying the past weekend was "intense" might have been a slight understatement! Just last weekend, the OWC 2017 Semifinals concluded, where 12 teams went in, and only 6 emerged from the ensuing battle. We have seen some truly magnificent and close matches, and some that displayed almost complete dominance from one side. Worry not if you missed all the fun – Worry not if you missed all the fun, we'll walk you through what happened in this week's recap!
+
+![](/wiki/shared/news/banners/owc_2017_recap_banner.jpg)
+
+Team Hong Kong never saw a shortage of talent. Despite being just a city, Hong Kong kept pumping out amazing players like [Chaoslitz](https://osu.ppy.sh/users/3621552), [DenierNezzar](https://osu.ppy.sh/users/126144), and [Ming3012](https://osu.ppy.sh/users/1583218) over the past years, to name a few. This year, Team Hong Kong ventured far into the OWC, taking down strong opponents left and right before making it to the front of Team Norway. 
+
+At this stage, the Semifinals mappool was putting players to the test, and their skills are being challenged from all directions. Team Hong Kong put up a good fight, but against Norwegian monsters like [CXu](https://osu.ppy.sh/users/84841), [-GN](https://osu.ppy.sh/users/895581), and [Afrodafro](https://osu.ppy.sh/users/3551255), they were denied the ticket to advance. 7-2 was the final score, but there were some seriously close matches that got decided by a matter of some thousands of points! The Norwegian players are anything but easy picking, and even they had to tip their hat to the Hong Kong beasts.
+
+![](/wiki/shared/news/2017-12-14-osu-world-cup-semifinals-recap/no_vs_hk.png)
```

Comment:
these images are fucking ginormous, drop res to 720p and jpeg 90, either or (or do both)